tests/test_taxi.py

```diff
@@ -1,81 +0,0 @@
-from django.contrib.auth import get_user_model
```

Comment:
why did you delete this file???

Reply:
I don't delete this file. I do: 
(venv) PS C:\Users\Nataly\py-taxi-service-startproject> git add taxi taxi_service .gitignore manage.py
(venv) PS C:\Users\Nataly\py-taxi-service-startproject> git commit -m 'Solution'

Reply:
do `git add .` and after thid do commit